clang/unittests/Interpreter/CMakeLists.txt

```diff
@@ -26,6 +26,13 @@ add_distinct_clang_unittest(ClangReplInterpreterTests
   TargetParser
   )
 
+if(NOT WIN32)
+  add_dependencies(ClangReplInterpreterTests
+    llvm-jitlink-executor
+    compiler-rt
```

Comment:
Clang does not depend on compiler-rt by default I'd think. Why not enabling this test only when we detect compiler-rt being on?